airsonic-main/src/main/java/org/airsonic/player/controller/LoginController.java

```diff
@@ -51,9 +51,11 @@ public ModelAndView login(HttpServletRequest request, HttpServletResponse respon
         map.put("brand", settingsService.getBrand());
         map.put("loginMessage", settingsService.getLoginMessage());
 
-        User admin = securityService.getUserByName(User.USERNAME_ADMIN);
-        if (User.USERNAME_ADMIN.equals(admin.getPassword())) {
-            map.put("insecure", true);
+        User admin = securityService.getUserByName("admin");
```

Comment:
Just because this is the only place it's ever used, and only once, and only because the initial install uses this value.  I could change it to a constant, but that kind of implies that it might be reasonable to use elsewhere, and it never should be.